examples/nxp/aot_neutron_compile.py

```diff
@@ -0,0 +1,323 @@
+        action="store_true",
+        required=False,
+        default=False,
+        help="Produce a quantized model",
+    )
+    parser.add_argument(
+        "-s",
+        "--so_library",
+        required=False,
+        default=None,
+        help="Provide path to so library. E.g., cmake-out/kernels/quantized/libquantized_ops_aot_lib.so. "
+        "To build it update the CMake arguments: -DEXECUTORCH_BUILD_KERNELS_QUANTIZED=ON"
+        " -DEXECUTORCH_BUILD_KERNELS_QUANTIZED_AOT=ON and build the quantized_ops_aot_lib package",
+    )
+    parser.add_argument(
+        "-p",
+        "--portable_lib",
+        required=True,
+        default=None,
+        help="Provide path to portable_lib so library. Should be in cmake-out/_portable_lib.cpython-310-x86_64-linux-gnu.so",
+    )
+    parser.add_argument(
+        "--debug", action="store_true", help="Set the logging level to debug."
+    )
+    parser.add_argument(
+        "-t",
+        "--test",
+        action="store_true",
+        required=False,
+        default=False,
+        help="Test the selected model and print the accuracy between 0 and 1.",
+    )
+    parser.add_argument(
+        "--operators_not_to_delegate",
+        required=False,
+        default=[],
+        type=str,
+        nargs="*",
+        help="List of operators not to delegate. E.g., --operators_not_to_delegate aten::convolution aten::mm",
+    )
+
+    args = parser.parse_args()
+
+    if args.debug:
+        logging.basicConfig(level=logging.DEBUG, format=FORMAT, force=True)
+
+    # 1. pick model from one of the supported lists
+    model, example_inputs, calibration_inputs = get_model_and_inputs_from_name(
+        args.model_name
+    )
+    model = model.eval()
+
+    # 2. Export the model to ATEN
+    exported_program = torch.export.export_for_training(
+        model, example_inputs, strict=True
+    )
+
+    # TODO: Add Neutron ATen Passes, once https://github.com/pytorch/executorch/pull/10579 is merged
+
+    module = exported_program.module()
+
+    # 4. Quantize if required
+    if args.quantize:
+        if args.quantize and (not args.so_library or not args.portable_lib):
+            logging.error(
+                "Quantization enabled without supplying path to libcustom_ops_aot_lib using --so_library and "
+                "_portable_lib.cpython* using --portable_lib CLI options. \n"
+                "This is required for running quantized models with unquantized input."
+            )
+            sys.exit(-1)
+        if calibration_inputs is None:
+            logging.warning(
+                "No calibration inputs available, using the example inputs instead"
+            )
+            calibration_inputs = example_inputs
+        module = post_training_quantize(module, calibration_inputs)
+
+        # For quantization we need to build the quantized_ops_aot_lib.so and _portable_lib.*.so
+        # Use this CMake options
+        # -DEXECUTORCH_BUILD_KERNELS_QUANTIZED=ON
```

Comment:
Is this documentation up to date? Is portable lib built just by specifying these two flags?

Reply:
FYI @skywall , we do not need any custom library loading for the quantized kernels out variants. There are already a python packages for this: 
```
import executorch.extension.pybindings.portable_lib
import executorch.kernels.quantized  # noqa F401
```
Thanks to @digantdesai for the review items which helped me to find it out.